invenio_deposit/views/ui.py

```diff
@@ -68,12 +76,37 @@ def index():
         return render_template(current_app.config['DEPOSIT_UI_INDEX_TEMPLATE'])
 
     @blueprint.route('/deposit/new')
+    @blueprint.route('/deposit/new/<string:deposit_type>')
```

Comment:
Btw, if you need this now, then add a ticket to fix it, and now that URLs on your site will change once it gets fixed.